src/applications/personalization/profile/components/direct-deposit/DirectDepositWrapper.jsx

```diff
@@ -0,0 +1,18 @@
+// Very simple component that decides which version of the DirectDeposit
+// component to show based on the value of the Flipper feature flag
+import React from 'react';
+
+import { connect } from 'react-redux';
+
+import { showDirectDepositV2 } from '../../selectors';
+import DirectDepositV1 from './DirectDeposit';
+import DirectDepositV2 from './DirectDepositV2';
+
+const DirectDepositWrapper = ({ showVersion2 }) => {
+  return <>{showVersion2 ? <DirectDepositV2 /> : <DirectDepositV1 />}</>;
```

Comment:
Do we actually need the Fragment here or can we just do

`const DirectDepositWrapper = ({ showVersion2 }) => showVersion2 ? <DirectDepositV2 /> : <DirectDepositV1 />;`

I guess adding the Fragment is in line with the file's name though 😄

Reply:
I'm pretty sure we actually do. I think this is where I was getting errors without the fragment. Oh I think I know a different way to write it...